python/lsst/obs/base/cameraMapper.py

```diff
@@ -815,8 +815,8 @@ def search(filename, description):
                 newPath = newPath[0] if newPath is not None and len(newPath) else None
                 if newPath is not None:
                     path = newPath
-            self.log.debug("Loading %s registry from %s", description, path)
             localFileObj = storage.getLocalFile(path)
+            self.log.info("Loading %s registry from %s", description, localFileObj.name)
```

Comment:
Commit message not entirely descriptive (since it also changes from path to name) but looks good.